packages/cli/src/commands/config/config.ts

```diff
@@ -1,9 +1,6 @@
-/**
- * @flow
- */
-import {type ConfigT} from 'types';
+import {Config} from '@react-native-community/cli-types';
 
-function isValidRNDependency(config) {
+function isValidRNDependency(config: any) {
```

Comment:
I think this config can be typed as:
```ts
config: AndroidDependencyConfig | IOSDependencyConfig | null
```

Reply:
I'm having trouble to set the type of this parameter, I try this approach but it's not working